auth/api/http/keys/transport.go

```diff
@@ -64,8 +72,13 @@ func decodeIssue(_ context.Context, r *http.Request) (interface{}, error) {
 }
 
 func decodeKeyReq(_ context.Context, r *http.Request) (interface{}, error) {
+	token := apiutil.ExtractBearerToken(r)
+	if strings.HasPrefix(token, patPrefix) {
+		return nil, apiutil.ErrUnsupportedTokenType
+	}
```

Comment:
This check is not required, Because in the above line 
`apiutil.ExtractBearerToken` have check like this `strings.HasPrefix(token, BearerPrefix)`
This will return empty string if token doesn't start with `Bearer`